Dockerfile

```diff
@@ -1,14 +1,15 @@
-FROM --platform=$BUILDPLATFORM golang:1.20-bookworm AS build
```

Comment:
What was the purpose of `--platform=$BUILDPLATFORM`? First time I see it. Was it unnecessary?

Reply:
the `--platform` forced to use the host os/arch even for cross-compiling. I think I ran into problems with this when compiling into CGO because the toolchain for compiling cgo on amd64 wasn't available in the arm64 golang image. Removing the `--platform=$BUILDPLATFORM` is effectively using arm64 image to build the arm64 binary and amd64 image to build the amd64 binary.

Reply:
Makes sense to me, thanks for the explanation.